dlpy/tests/test_model.py

```diff
@@ -90,7 +90,7 @@ def test_model1(self):
                                casout={'name': 'eee', 'replace': True},
                                path=path)
 
-        r = model1.fit(data='eee', inputs='_image_', target='_label_', lr=0.001)
+        r = model1.fit(data='eee', inputs='_image_', target='_label_', lr=1e-5)
```

Comment:
This value is used in lots of places. Should it be defined as a module or package variable?

Reply:
I defined safe_learning_rate as a class variable for this test in my latest update.  I reran this unit test and everything still seems stable.